### ISSUE TITLE
New travis problem

### ISSUE THREAD

**Issue by mjohnson541** [ReactionMechanismGenerator/RMG-Py#1208]
creating build/build/pyrex/rmgpy/kinetics

x86_64-conda_cos6-linux-gnu-gcc -pthread -fno-strict-aliasing -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O3 -pipe -DNDEBUG -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I. -I/home/travis/miniconda/envs/rmg_env/lib/python2.7/site-packages/numpy/core/include -I/home/travis/miniconda/envs/rmg_env/include/python2.7 -c build/pyrex/rmgpy/kinetics/arrhenius.c -o build/build/pyrex/rmgpy/kinetics/arrhenius.o

unable to execute 'x86_64-conda_cos6-linux-gnu-gcc': No such file or directory

error: command 'x86_64-conda_cos6-linux-gnu-gcc' failed with exit status 1

make: *** [main] Error 1

The command "make" failed and exited with 2 during .

Your build has been stopped.

**Comment by mliu49** [comment]
I think this might have been some issue with miniconda, which they seem to have fixed since new builds are no longer encountering this error.

However, there is a new test failure which is occurring on multiple branches, which I have been unable to reproduce locally.
```
======================================================================
FAIL: This tests to ensure that a usable Cantera input file is created.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/rmg/mainTest.py", line 176, in testMakeCanteraInputFile
    self.fail('The output Cantera file is not loadable in Cantera.')
AssertionError: The output Cantera file is not loadable in Cantera.
----------------------------------------------------------------------
```
Raising the error gives the following:
```
======================================================================
ERROR: This tests to ensure that a usable Cantera input file is created.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/ReactionMechanismGenerator/RMG-Py/rmgpy/rmg/mainTest.py", line 174, in testMakeCanteraInputFile
    ct.Solution(os.path.join(outName, f))
  File "interfaces/cython/cantera/base.pyx", line 26, in cantera._cantera._SolutionBase.__cinit__ (interfaces/cython/cantera/_cantera.cpp:7060)
  File "interfaces/cython/cantera/base.pyx", line 62, in cantera._cantera._SolutionBase._init_cti_xml (interfaces/cython/cantera/_cantera.cpp:7596)
RuntimeError: 
***********************************************************************
CanteraError thrown by addElementsFromXML:
no data for element /2.014/
***********************************************************************
```

**Comment by mliu49** [comment]
Ok, so it turned out to be an issue with the Cantera version. For some reason, the Travis builds are installing 2.3.0a2, while the most recent packages in the RMG channel is 2.3.0a3. The new version fixes the issue. I have no idea why this issue just started appearing recently.

It seems that creating a new environment from the environment files uses 2.3.0a2 by default (verified locally). So why does it not use 2.3.0a3? The most likely explanation is dependency conflicts, although it's hard to pinpoint the exact issue. Here is what conda says if you try to upgrade from 2.3.0a2 to 2.3.0a3:
```
The following packages will be UPDATED:

    cantera: 2.3.0a2-py27_0        rmg --> 2.3.0a3-np111py27_4_g89d0c56 rmg
    numpy:   1.10.2-py27_0             --> 1.11.3-py27_0                   
    rdkit:   2015.09.2-np110py27_0 rmg --> 2016.03.4-np111py27_1        rmg

The following packages will be DOWNGRADED:

    mkl:     2018.0.0-hc285769_4       --> 2017.0.3-0                      
    scipy:   0.19.1-py27hf01dd8f_3     --> 0.19.0-np111py27_0  
```

However, it doesn't try to change any dependencies if you downgrade to 2.3.0a2, which suggests that 2.3.0a3 was built with stricter dependency requirements. One possible solution would be to require version 2.3.0a3 in the environment file. Thoughts and suggestions are welcome.

**Comment by mliu49** [comment]
It seems that the conda packages for mkl, numpy, and scipy were updated 5 days ago, which coincides with the start of our problems...

We are probably installing cantera 2.3.0a2 and rdkit 2015 because conda is choosing to install numpy 1.10. My guess is that mkl 2018 and scipy 0.19.1 require numpy 1.10 for some reason.

As can be seen on the `cantera_requirement` branch, enforcing cantera 2.3.0a3 solves the issue.

**Comment by mliu49** [comment]
Fixed by #1212.
